pandas/io/formats/style.py

```diff
@@ -3133,8 +3144,12 @@ def highlight_null(
         def f(data: DataFrame, props: str) -> np.ndarray:
             return np.where(pd.isna(data).to_numpy(), props, "")
 
+        if null_color != "red":
+            raise FutureWarning("`null_color` is deprecated: use `color` instead")
```

Comment:
Shouldn't this use `warning.warn(...)`?

Also needs a whatsnew note for 1.5.0

Reply:
agreed chged